src/server/verification/verificationAPI.js

```diff
@@ -466,6 +468,11 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
           log.error('Bonuses charge failed', e.message, e, user)
 
           fail()
+
+          return res.status(400).json({
```

Comment:
you need to check a response was not already sent by ontxhash

Reply:
done